core/iwasm/common/wasm_runtime_common.c

```diff
@@ -6148,3 +6152,19 @@ wasm_runtime_get_context(WASMModuleInstanceCommon *inst, void *key)
     return wasm_native_get_context(inst, key);
 }
 #endif /* WASM_ENABLE_MODULE_INST_CONTEXT != 0 */
+
+#if WASM_ENABLE_AOT != 0
```

Comment:
Is it better if merge JIT perf support operations here? w/o using a member in `LLVMJITOptions`?

Reply:
Yes, had better just merge them into one option here. And had better introduce a cmake variable (e.g. WAMR_BUILD_LINUX_PERF_SUPPORT) and compiler macro (e.g. WASM_ENABLE_LINUX_PERF_SUPPORT) for this feature. Make it configurable so as not to increase footprint when it isn't enabled.